_data/events_gdcr2019/poznan.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "GDCR Poznan",
+  "url": "https://gskit.pl/",
```

Comment:
Hi @soycapitan! Thanks for letting us know about your event. Please, we'd need an `url` for the event. It can be a Meetup page, a webpage about the event, an Eventbrite page, ... Any page where an attendee can find the information about your event and know how to register. You can see examples on [the included events](https://www.coderetreat.org/events/) until now. When you have the `url` of that page, please edit the file to include it and push it again in the same branch. Don't worry. We'll keep this pull request open until you have that url. Thanks!

Reply:
@rachelcarmena 
Thank you. Your remark is acknowledged. I will work with the team to have some webpage setup and/or entry on eventbrite created.

Reply:
Great! Thanks to you, @soycapitan!

Reply:
Done, please recheck.

Reply:
Great!! Thanks @soycapitan!!